src/core/SmartWeaveTags.ts

```diff
@@ -15,6 +15,5 @@ export enum SmartWeaveTags {
   INTERACT_WRITE = 'Interact-Write',
   WASM_LANG = 'Wasm-Lang',
   WASM_LANG_VERSION = 'Wasm-Lang-Version',
-  CONTRACT_TYPE = 'Contract-Type',
```

Comment:
@asiaziola , are using this tag somewhere in SonAR?

Reply:
@ppedziwiatr no.